app/views/bookings/show.html.erb

```diff
@@ -0,0 +1,21 @@
+<div class="container">
+  <div class="row text-center">
+    <h1>Your booking</h1>
+  </div>
+  <div class="col-xs-4 col-xs-offset-4 text-center">
+    <div class="booking-date-infos">
+      <p>Starts on: <%= @booking.starts_at %></p>
+      <p>Ends on: <%= @booking.ends_at %></p>
+      <p>Status: <%= @booking.status %></p>
+    </div>
+    <div class="booking-car-infos">
+      <%= link_to 'My PimpedCar', pimped_car_path %>
```

Comment:
doesn't link to the good pimped car (the id you pass is the booking id and not the car id)
try : pimped_car_path(@booking.pimped_car_id)